backend/app/tests/crud/collections/collection/test_crud_collection_delete.py

```
@@ -4,29 +4,19 @@
 from app.crud import CollectionCrud
 from app.models import APIKey, Collection, ProviderType
-from app.crud.rag import OpenAIAssistantCrud
+from app.crud.rag import OpenAIVectorStoreCrud
 from app.tests.utils.utils import get_project
 from app.tests.utils.document import DocumentStore
 
 
-def get_assistant_collection_for_delete(
-    db: Session, client=None, project_id: int = None
+def get_vector_store_collection(
+    db: Session, client: OpenAI, project_id: int
```

Ruff (0.15.12)

[warning] Unused function argument: `db`

(ARG001)